docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -1,17 +1,2133 @@
+
+Default: `""`
+
+|`font`
+|`style`
+|Font settings. Technically, you can put other styles in here too 
+
+Default: `{<<font_fn>>}`
+|===
+
+*Returns:* `render`
+
+
+[float]
+[[math_fn]]
+=== `math`
+
+Interprets a math expression, with a `number` or `datatable` as context. Datatable 
+columns are available by their column name. If you pass in a `number`, it is 
+available as `"value"` (without the quotes).
+
+*Accepts:* `number`, `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `expression`
+|`string`
+|An evaluated TinyMath expression, see <<canvas-tinymath-functions>>
+|===
+
+*Returns:* `number`
+
+
+[float]
+[[metric_fn]]
+=== `metric`
+
+A number with a label.
+
+*Accepts:* `number`, `string`, `null`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_
+|`string`
+|Text describing the metric 
+
+Default: `""`
+
+|`metricFont`
+|`style`
+|Font settings for the metric. Technically, you can put other styles in here too.
+
+Default: `{<<font_fn>> size=48 family="'Open Sans', Helvetica, Arial, sans-serif" color="#000000" align=center lHeight=48}`.
+
+|`labelFont`
+|`style`
+|Font settings for the label. Technically you can put other styles in here too.
+
+Default: `{<<font_fn>> size=14 family="'Open Sans', Helvetica, Arial, sans-serif" color="#000000" align=center}`.
+|===
+
+*Returns:* `render`
+
+
+[float]
+[[n_fns]]
+== N
+[[neq_fn]]
+=== `neq`
+
+Returns whether the _context_ is not equal to the argument.
+
+*Accepts:* `boolean`, `number`, `string`, `null`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `value`
+|`boolean`, `number`, `string`, `null`
+|The value to compare the _context_ to
+|===
+
+*Returns:* `boolean`
+
+
+[float]
+[[p_fns]]
+== P
+[[palette_fn]]
+=== `palette`
+
+Creates a color palette
+
+*Accepts:* `null`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** *†*
+
+Alias: `color`
+|`string`
+|Palette colors, rgba, hex, or HTML color string
+
+|`gradient`
+|`boolean`
+|Make a gradient, where supported 
+
+Default: `false`
+
+|`reverse`
+|`boolean`
+|Reverse the palette 
+
+Default: `false`
+|===
+
+*Returns:* `palette`
+
+
+[float]
+[[pie_fn]]
+=== `pie`
+
+Configures a pie chart element.
+
+*Accepts:* `pointseries`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|`palette`
+|`palette`
+|A palette object for describing the colors to use on this pie 
+
+Default: `{<<palette_fn>>}`
+
+|`seriesStyle` *†*
+|`seriesStyle`
+|A style of a specific series
+
+|`radius`
+|`string`, `number`
+|Radius of the pie as a percentage (between 0 and 1) of the available space. 
+Set to `"auto"` to automatically set radius 
+
+Default: `"auto"`)
+
+|`hole`
+|`number`
+|Draw a hole in the pie, 0-100, as a percentage of the pie radius 
+
+Default: `0`
+
+|`labels`
+|`boolean`
+|Show pie labels 
+
+Default: `true`
+
+|`labelRadius`
+|`number`
+|The percentage of the area of container to use as a radius for the label circle 
```

Comment:
How about:

The percentage of the container area to use as a radius for the label circle.